auth-server/src/main/java/com/fiftyfive/authserver/configs/KeycloakConfig.java

```diff
@@ -0,0 +1,34 @@
+package com.fiftyfive.authserver.configs;
+
+import org.keycloak.adapters.springboot.KeycloakSpringBootConfigResolver;
+import org.keycloak.adapters.springsecurity.KeycloakConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.oauth2.jwt.JwtDecoder;
+import org.springframework.security.oauth2.jwt.JwtDecoders;
+
+@KeycloakConfiguration
+public class KeycloakConfig {
+
+    /**
+     * Use properties in application.properties instead of keycloak.json
+     */
+    @Bean
+    public KeycloakSpringBootConfigResolver keycloakConfigResolver() {
+
+        return new KeycloakSpringBootConfigResolver();
+    }
+
+    /**
+     * JwtDecoder helps in decoding ,verifying and validating jwt
+     */
+    @Bean
+    public JwtDecoder jwtDecoder() {
+        /**
+         * Create a JwtDecoder using the Keycloak issuer URL and realm
+         */
+        String issuerUrl = "http://localhost:8180/realms/userdev";
```

Comment:
should have been declared in application.properties file

Reply:
okay,sure